src/main/java/ch/cern/eam/wshub/core/services/workorders/entities/Activity.java

```diff
@@ -80,6 +80,8 @@
 	@Transient
 	private WorkOrderActivityChecklistSignatureResult[] signatures;
 
+	private Boolean forceActivityExpansion;
```

Comment:
`@Transient` annotation missing.

Explanation: with the use of JPA, this class is mapped to the R5ACTIVITIES table (`@Table(name = "R5ACTIVITIES")`). Fields are then mapped to columns with the `@Column` annotation. If `@Column` is not specified, it will by default map it to a column with a name based on the field name (`forceActivityExpansion`). To disable this behavior and not map the field to a database column, the `@Transient` annotation can be used.